core/state_processor_test.go

```diff
@@ -374,7 +374,8 @@ func TestProcessVerkle(t *testing.T) {
 		}
 		signer     = types.LatestSigner(config)
 		testKey, _ = crypto.HexToECDSA("b71c71a67e1177ad4e901695e1b4b9ee17ae16c6668d313eac2f96dbcda3f291")
-		db         = rawdb.NewMemoryDatabase()
+		bcdb       = rawdb.NewMemoryDatabase() // Database for the blockchain
+		gendb      = rawdb.NewMemoryDatabase() // Database for the block-generation code, they must be separate as they are path-based.
```

Comment:
Since we have this "reorg-unfriendly" PBSS, I can not use the same DB for generating the blocks and inserting them in the blockchain: in one instance, it tries to create a contract over an already existing one, and that causes all the gas to be consumed. So I use two databases here.